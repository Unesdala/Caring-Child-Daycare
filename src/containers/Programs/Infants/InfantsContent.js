import React from 'react';

const InfantsContent = () => (
  <main className="infants">
    <p />
  </main>
);

export default InfantsContent;
