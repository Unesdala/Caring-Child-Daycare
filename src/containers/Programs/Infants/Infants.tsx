import InfantsContent from './InfantsContent';

export const Infants = (): JSX.Element => {
  document.title = 'Caring Child Daycare | Infants & Toddlers';  
  return (
    <div className="Site-content">
      <main className="infants-main">
        <InfantsContent />
      </main>
    </div>
  );
};

export default Infants;
