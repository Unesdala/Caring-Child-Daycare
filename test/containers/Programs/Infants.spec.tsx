/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Infants } from 'src/containers/Programs/Infants/Infants';
import InfantsContent from 'src/containers/Programs/Infants/InfantsContent';

describe('Infants', () => {
  const wrapper = shallow(<Infants />);
  it('renders', () => {
    expect(wrapper.find(InfantsContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(InfantsContent).dive().find('main.infants-main--content').exists()).toBe(true);
  });
  
});
