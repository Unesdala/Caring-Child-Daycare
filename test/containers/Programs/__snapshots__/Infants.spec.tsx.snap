// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Infants renders 1`] = `ShallowWrapper {}`;
